packages/common/src/types/document.ts

```diff
@@ -11,5 +11,18 @@ export type TDocument = {
   document: Record<string, TDocumentField>;
 };
 
-// Document history can return TDocumentRecord[]
 export type TDocumentRecord = TDbRecord<TDocument>;
+
+export type TDocumentHistory = {
+  docId: string;
+  documents: TDocumentRecord[];
+  metadata: TDocumentHistoryMetadata;
+  active: boolean;
+};
+
+export type TDocumentHistoryMetadata = {
```

Comment:
This should be defined in `packages/common/src/types/metadata.ts`

Reply:
fixed by using the TMetadataDetail